prod-aos-rcar.yaml

```diff
@@ -0,0 +1,357 @@
+desc: "Xen-Troops development setup for Renesas RCAR Gen3 hardware"
+min_ver: "0.8"
+
+variables:
+  YOCTOS_WORK_DIR: "yocto"
+  DOM0_BUILD_DIR: "build-dom0"
+  DOMD_BUILD_DIR: "build-domd"
+  DOMF_BUILD_DIR: "build-domf"
+  XT_DOMD_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-domd.dtb"
+  XT_XEN_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-xen.dtb"
+  XT_OP_TEE_FLAVOUR: "generic_dt"
+  GENERIC_MACHINE: "generic-armv8-xt"
+
+common_data:
+  # Sources used by all yocto-based domains
+  sources: &COMMON_SOURCES
+    - type: git
+      url: "git://git.yoctoproject.org/poky"
+      rev: dunfell
+    - type: git
+      url: "git://git.openembedded.org/meta-openembedded"
+      rev: dunfell
+    - type: git
+      url: "git://git.yoctoproject.org/meta-virtualization"
+      rev: dunfell
+    - type: git
+      url: "https://github.com/xen-troops/meta-xt-common.git"
+      rev: master
+    - type: git
+      url: "https://github.com/xen-troops/meta-xt-rcar.git"
+      rev: master
+    - type: git
+      url: "https://github.com/al1img/meta-xt-prod-aos-rcar.git"
+      rev: aos_integration
```

Comment:
change at xen-troops and branch 'main'

Reply:
done

Reply:
URL still points to personal repo `github.com/al1img`.